bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/handler/GardenaAccountHandler.java

```diff
@@ -158,9 +256,12 @@ public Collection<Class<? extends ThingHandlerService>> getServices() {
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
         if (RefreshType.REFRESH == command) {
+            // ****
+            // TODO: REALLY ??? should a UI refresh be causing a complete initialisation of the bridge ???
```

Comment:
Pfew, no, definitely not. Btw, it is not a "UI refresh", but a command that can be sent e.g. by rules. As it is also called once for each channel during startup, this code here can potentially cause quite some hiccups during system startup, so I would tend to completely remove it, if we do not have a chance to easily provide an up-to-date value for the requested channel.

Reply:
Ok. Thanks for the confirmation. I will remove it.